staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -79,18 +79,18 @@ func (t *azureDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Vol
 					Driver:           AzureDiskDriverName,
 					VolumeHandle:     azureSource.DataDiskURI,
 					ReadOnly:         *azureSource.ReadOnly,
-					FSType:           *azureSource.FSType,
 					VolumeAttributes: map[string]string{azureDiskKind: "Managed"},
 				},
 			},
 			AccessModes: []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce},
 		},
 	}
 
-	if *azureSource.CachingMode != "" {
+	if azureSource.CachingMode != nil {
```

Comment:
to preserve same behavior I think we still want to check for `len==0` case and not fill in an empty string in the `VolumeAttribute`